SELearning.API/Controllers/CommentController.cs

```diff
@@ -82,5 +117,59 @@ public async Task<IActionResult> UpdateComment(int ID, CommentDTO comment)
     [ProducesResponseType(204)]
     [ProducesResponseType(404)]
     public async Task<IActionResult> DeleteComment(int ID)
-        => (await _repository.DeleteAsync(ID)).ToActionResult();
+    {
+        try
+        {
+            await _service.RemoveComment(ID);
+            return NoContent();
+        }
+        catch (CommentNotFoundException)
+        {
+            return NotFound();
+        }
+    }
+
+    /// <summary>
+    /// <c>UpvoteComment</c> increases the rating of the comment with the given ID.
+    /// </summary>
+    /// <param name="ID">The ID of the comment.</param>
+    /// <returns>A response type 204: No Content if the comment exists, otherwise response type 404: Not Found.</returns>
+    [Authorize]
+    [HttpPut("{ID}")]
```

Comment:
@JoachimBorup This will resolve to the same endpoint as update, Should probably be {ID}/upvote